File: C#/Remove Duplicates From Sorted List/RemoveDuplicates.cs
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     public int val;
 *     public ListNode next;
 *     public ListNode(int x) { val = x; }
 * }
 */
public class Solution {
    public ListNode DeleteDuplicates (ListNode head) {
        if (head == null) return head;
        ListNode cursor = head;
        while (cursor.next != null) {
            if (cursor.val == cursor.next.val)
                cursor.next = cursor.next?.next;
            else
                cursor = cursor.next;
        }
        return head;
    }
}

File: Python/Remove Duplicates From Sorted List/RemoveDuplicates.py
# Definition for singly-linked list.
# class ListNode:
#     def __init__(self, x):
#         self.val = x
#         self.next = None


class Solution:
    def deleteDuplicates(self, head):
        """
        :type head: ListNode
        :rtype: ListNode
        """
        if (head):
            cursor = head
            while (cursor.next):
                if (cursor.next.val != cursor.val):
                    cursor = cursor.next
                else:
                    cursor.next = cursor.next.next
        return head